dockerfiles/theia/src/get-webview-route.js

```diff
@@ -0,0 +1,73 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+const fs = require('fs');
+const workspaceClient = require('@eclipse-che/workspace-client');
```

Comment:
where does it find this dependency ?

Reply:
It gets the dependency from Che Theia node modules.

Reply:
In the dockerfile we copy script to the /home/theia, where is located node_modules